webapp/web-components/cht-form/tests/karma/stubs/cht-datasource.service.spec.ts

```diff
@@ -0,0 +1,53 @@
+import sinon from 'sinon';
+import { SinonStub } from 'sinon';
+import { expect } from 'chai';
+import { TestBed } from '@angular/core/testing';
+import { CHTDatasourceService } from '../../../src/stubs/cht-datasource.service';
+
+describe('CHT Datasource Service', () => {
+  let service: CHTDatasourceService;
+  let consoleError: SinonStub;
+
+  beforeEach(() => {
+    consoleError = sinon.stub(console, 'error');
+    service = TestBed.inject(CHTDatasourceService);
+  });
+
+  afterEach(() => sinon.restore());
+
+  it('getSync', () => {
+    const chtApi = service.getSync();
+
+    expect(chtApi).to.have.property('v1');
```

Comment:
I prefer to be exclusive here instead of inclusive, and go for something like: 
```suggestion
expect(chtApi).to.have.keys(['v1']);
expect(chtApi.v1).to.have.keys(['getExtensionLib']);
```